tasks/managed/add-fbc-contribution/add-fbc-contribution.yaml

```diff
@@ -49,10 +49,12 @@ spec:
       description: The targetIndex used in this request
     - name: requestResultsFile
       description: Internal Request results file
+    - name: internalRequestResultsFile
+      description: Additional Internal Request results file
     - name: requestMessage
       description: Internal Request message
     - name: requestReason
-      description: Internal Request reason
+      description: Internal Request reason      
```

Comment:
typo? doesn't seem to be a reason to add the whitespace

Reply:
fixed.